### PR TITLE
Interface tweaks

### LINKED ISSUE #60: Navigation
great job making the reserve amenity button specific to the amenity (eg reserve bbq in zone c) , and then showing the confirmation message “amenity reserved” - the confirmation message could also point people to see their reservations in the My Reservations section and that could possibly be linked as they might not have seen that in the menu yet if they didn’t expand the mini drawer ui

----

Let's also make a button to make another reservation

### LINKED ISSUE #70: minor formatting issue on email confirmation page
both bits of text should be centered 
image

### PR DESCRIPTION
**Pull request by djensenius**
Add reserve again to reservation confirmation.
Add link to your bookings to amenity confirmation.
Center some thank you texts.

Closes #70 
Closes #60

### CODE REVIEW

File: src/Reservation.tsx [hunk 1]
```diff
@@ -327,6 +335,28 @@ export default function Resevation(): JSX.Element {
...
+                variant="contained"
+                className={classes.registerButton}
+                onClick={(): void => setThanks(false)}
+                startIcon={<EventAvailable />}
+                type="submit"
+              >
+                Make Another Reservation
+              </Button>
+            </Grid>
+            <Grid item xs={12}>
```
Comment (MaggieShan) [inline src/Reservation.tsx:349]: Could this button also be centred?

Comment (MaggieShan) [reply src/Reservation.tsx:349]: In doing so, it might also look nicer by decreasing the grid container's spacing

Comment (djensenius) [reply src/Reservation.tsx]: Great feedback! Addressed!

**Comment by MaggieShan** [comment]
After clicking reserve again, it seems like the the form is not reset, and so I get a few undesired behaviours:  
- default start/end times are the same as when first inputted (even when date changes)
- amenity alert shows the reservation times for the previously chosen amenity 
- there is no error checking - so I'm able to reserve the same amenity at the same time
Screen Shot 2020-08-31 at 8 45 28 PM

**Comment by djensenius** [comment]
I was not able to reserve the amenity for the same time.

I got rid of the availability when making a new reservation.

Times should all be reset to now (or nearest 15 mins when that's done 😊)

**Comment by MaggieShan** [comment]
Hmm I'm still able to reserve the amenity at the same time. Try these steps: 
1. Reserve an amenity at a valid time 
2. Select [Make Another Reservation] 
3. Select the same amenity, check the question and [Reserve] 
(I've noticed that if the clock was touched, then an error message appears, but if not, it will allow it) 

Also, sorry I'm hoping to get that done soon!

**Comment by djensenius** [comment]
Hmmm. Is the first reservation or the second reservation just the default times (begins and ends at the same minute)?

If so, that won't trigger the overlap Postgres thing.

**Comment by MaggieShan** [comment]
Ya they are! Ooh okay that makes sense, It seems like the overlap operator counts dates that are equal as not overlapping.

**Comment by djensenius** [comment]
I think it's okay since it only happens with one minute reservations.

If it's a bigger issue let's create a new story!

**Pull request merged**